server/athenian/api/internal/miners/jira/issue.py

```diff
@@ -769,6 +836,16 @@ async def query_jira_raw(
     if jira_filter.status_categories:
         and_filters.append(Status.category_name.in_(jira_filter.status_categories))
 
+    if mapped_to_prs is not None:
+        assert meta_ids
+        PRIssues = NodePullRequestJiraIssues
+        subselect = sa.select(PRIssues.jira_id).where(
+            PRIssues.jira_acc == jira_filter.account,
+            PRIssues.node_acc.in_(meta_ids),
+            PRIssues.node_id.progressive_in(mapped_to_prs),
+        )
+        and_filters.append(Issue.id.in_(subselect))
+
```

Comment:
At the line
```python
 q.with_statement_hint("Leading(((athenian_issue issue) (s c)))")
```
we should check if we've got `mapped_to_prs` and change `Leading` to `(((PRIssues.__tablename__ *VALUES*) athenian_issue) issue)) (s c)` or `((PRIssues.__tablename__ athenian_issue) issue)) (s c)` depending on `progressive`.